### DISCUSSION TITLE
Why this is not working for me?

### DISCUSSION THREAD

**Discussion by suitnes** [vslinko/obsidian-zoom#55] [Q&A]
![Desktop screenshot](https://user-images.githubusercontent.com/97612486/152256733-2e11c165-bbbd-4d7e-93cb-7be60fafc4a0.png)

Hi. I'm new here.
I istalled everything according to your instructions in the Read Me section, I installed it from obsidian, but it doesn't work, then installed it using the other mode.
But the lines are not working. What will I be doing wrong?

**Comment by vslinko** [comment]
Hi @suitnes ,

Outliner plugin is only works with [markdown lists](https://www.markdownguide.org/basic-syntax/#lists-1).

Please write a dash and a space `- ` before you write the text. This should enable the Outliner plugin for this list.
